components/globals/AdminNav.tsx

```diff
@@ -48,7 +48,13 @@ const AdminNav = (props: AdminNavProps): React.ReactElement => {
             </Link>
           )}
           {user && user.name && (
-            <button className="gc-button-link" onClick={() => signOut()}>
+            <button
+              className="gc-button-link"
+              onClick={() => {
+                sessionStorage.clear();
```

Comment:
Thoughts on scoping this to just `form-storage` ?

Reply:
Probably better to be scoped yes. I'll use the clearStorage function